### PR TITLE
SCLI-4 Reduce Timeout and Re-connection Duration

### PR DESCRIPTION
**Pull request by metinavc1**

### CODE REVIEW

File: Library/Network/SPIWebSocketConnection.m [hunk 1]
```diff
@@ -42,15 +42,15 @@ - (void)connect {
     
     // Create a new socket instance specifying the url, SPI protocol and Websocket to use.
     // The will create a TCP/IP socket connection to the provided URL and perform HTTP websocket negotiation
-    self.webSocket          = [[SRWebSocket alloc] initWithURLRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:self.url] cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:8.0] protocols:@[@"spi.2.3.0"]];
+    self.webSocket          = [[SRWebSocket alloc] initWithURLRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:self.url] cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:4.0] protocols:@[@"spi.2.3.2"]];
```
Comment (gouline) [inline Library/Network/SPIWebSocketConnection.m]: * Read what it says, you've just changed the cache policy, not the timeout
* Protocol version doesn't change

File: Library/Network/SPIWebSocketConnection.m [hunk 2]
```diff
@@ -42,15 +42,15 @@ - (void)connect {
...
     // Let's let our users know that we are now connecting...
     self.state = SPIConnectionStateConnecting;
     [self.webSocket open];
     
     // We have noticed that sometimes this websocket library, even when the network connectivivity is back,
-    // it never recovers nor gives up. So here is a crude way of timing out after 8 seconds.
+    // it never recovers nor gives up. So here is a crude way of timing out after 4 seconds.
     __weak __typeof(&*self) weakSelf = self;
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
         sleep(8);
```
Comment (gouline) [inline Library/Network/SPIWebSocketConnection.m]: This is the timeout!

**Review by gouline — CHANGES_REQUESTED** [review]

File: Library/Network/SPIWebSocketConnection.m (hunk 1 above)
Comment (metinavc1) [reply Library/Network/SPIWebSocketConnection.m]: fixed

File: .gitignore [hunk 3]
```diff
@@ -29,3 +29,4 @@ xcuserdata/
 *.dSYM
 
 build.log
+*.DS_Store
```
Comment (gouline) [inline .gitignore]: This doesn't belong here, you need to configure your global gitignore to include it. Revert.

**Review by gouline — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (metinavc1) [reply .gitignore]: Done

**Comment by gouline** [comment]
@metinavci Please update the version in podspec before I can approve this PR. Thanks.

**Review by gouline — APPROVED** [review]

**Pull request merged**